accounts/migrations/0006_auto_20210113_2249.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.1.3 on 2021-01-13 22:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '0005_auto_20201026_1312'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='customuser',
+            name='about',
+            field=models.TextField(default='', help_text='A short description of yourself'),
+        ),
+        migrations.AddField(
+            model_name='customuser',
+            name='full_name',
+            field=models.CharField(default='', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='customuser',
+            name='website_url',
+            field=models.URLField(default='', help_text='Website, GitHub or Linkedin URL.', max_length=255),
+        ),
+        migrations.AlterField(
+            model_name='customuser',
+            name='current_lms_module',
+            field=models.CharField(choices=[('', 'Select Learning Stage'), ('programme_preliminaries', 'Programme Preliminaries'), ('programming_paradigms', 'Programming Paradigms'), ('html_essentials', 'HTML Essentials'), ('css_essentials', 'CSS Essentials'), ('user_centric_frontend_development', 'User Centric Frontend Development'), ('comparative_programming_languages_essentials', 'Comparative Programming Languages Essentials'), ('javascript_essentials', 'Javascript Essentials'), ('interactive_frontend_development', 'Interactive Frontend Development'), ('python_essentials', 'Python essentials'), ('practical_python', 'Practical Python'), ('data_centric_development', 'Data Centric Development'), ('backend_development', 'Backend Development'), ('full_stack_frameworks_with_django', 'Full Stack Frameworks with Django'), ('alumni', 'Alumni'), ('staff', 'Staff')], default='', max_length=50),
```

Comment:
Is this line PEP8 compliant? Can it be cleaned-up a bit, or will that hinder readability?

Reply:
This is a migration so it's auto generated. I made it PEP8 compliant though.

Reply:
Oh wow! I normally don't review the migration files, not sure how this one slipped by me as being a migration file.
Sorry Stefan! Could've ignored me and left as is...